packages/frontend/src/state/positions/providers.tsx

```diff
@@ -103,15 +104,16 @@ export const ComputeSwapsProvider: FC = ({ children }) => {
     } else setPositionType(PositionType.NONE)
   }, [computedSwaps.squeethAmount, oSqueethBal, setPositionType])
```

Comment:
Also should we change to useAppEffect for this?

Reply:
I can see you worked on it but let me handle it on my PR.

Reply:
Basically, if the dependency contains a big number, we should use useAppEffect. Other than that, useEffect is fine.

Reply:
yh I mean if we dont change to useAppEffect, it may cause other loading issue since Bignumber as a object in dep array created lots of issue before